tmpreaper.te

```diff
@@ -59,10 +59,7 @@ fs_delete_tmpfs_files(tmpreaper_t)
 files_read_var_lib_files(tmpreaper_t)
 files_purge_tmp(tmpreaper_t)
 files_delete_all_non_security_files(tmpreaper_t)
-# why does it need setattr?
-files_setattr_all_tmp_dirs(tmpreaper_t)
```

Comment:
should be checked if these settattr interfaces calls are really covered by non_security_file_type